Libplanet/Net/Swarm.BlockSync.cs

```diff
@@ -737,9 +740,10 @@ CancellationToken cancellationToken
                         blockChain.Id,
                         synced.Id
                     );
+                    bool render = !syncedB.Id.Equals(forked.Id);
```

Comment:
Looks like `forked` is already deleted when catching the exception.

```
            catch (Exception)
            {
                _store.DeleteChainId(forked.Id);
                FillBlocksAsyncFailed.Set();
                throw;
            }
```

is it ok to access `forked.Id` here?

Reply:
In short thoughts, it seems the `synced is { } syncedB` condition checks it is processed without throwing exception. 🤔 

https://github.com/planetarium/libplanet/blob/c8c056acdc3f2e52cd4973be93e2fb993bfed81d/Libplanet/Net/Swarm.BlockSync.cs#L726-L731